mitodl/ocw-studio
# add UrlAssembler package

#### Pre-Flight checklist



#### What are the relevant tickets?

none

#### What's this PR do?

Just adds a nicety, a package called url-assembler that we used for building up URLs in open-discussions.

#### How should this be manually tested?

check all the frontend functionality to make sure it is all working as intended. read through the code and make sure the changes all make sense.
